### PR TITLE
Go: Convert log-injection, path-injection and command-injection sinks to use models-as-data

### PR DESCRIPTION
**Pull request by owen-mc**
Convert log-injection, path-injection and command-injection sinks to use models-as-data. Follows on from https://github.com/github/codeql/pull/17072.

### CODE REVIEW

File: go/ql/lib/ext/github.com.sirupsen.logrus.model.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
+extensions:
+  - addsTo:
+      pack: codeql/go-all
+      extensible: packageGrouping
+    data:
+      - ["logrus", "github.com/sirupsen/logrus"]
+      - ["logrus", "ggithub.com/Sirupsen/logrus"]
```
Comment (egregius313) [inline go/ql/lib/ext/github.com.sirupsen.logrus.model.yml:7]: Typo in package path
```suggestion
      - ["logrus", "github.com/Sirupsen/logrus"]
```

**Review by egregius313 — CHANGES_REQUESTED** [review]
LGTM other than typo in logrus package path

**Comment by owen-mc** [comment]
@egregius313 Thanks for spotting that! I accepted your suggested change. Please re-review.

**Review by egregius313 — APPROVED** [review]
LGTM

**Pull request merged**